sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs

```diff
@@ -43,14 +43,16 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("Compute", "VirtualMachineExtensions", "2021-03-01"),
                 new Tuple<string, string, string>("Compute", "VirtualMachineImages", "2021-03-01"),
                 new Tuple<string, string, string>("Compute", "VirtualMachineImagesEdgeZone", "2021-03-01"),
-                new Tuple<string, string, string>("Compute", "VirtualMachineRunCommands", "2021-03-01"),
+                new Tuple<string, string, string>("Compute", "VirtualMachineRunCommands", "2020-12-01"),
```

Comment:
This is incorrect. You might be based off an older swagger branch. Undo this.

Reply:
fixed